app/src/main/java/org/jellyfin/mobile/utils/SystemUtils.kt

```diff
@@ -115,6 +118,12 @@ private fun Context.downloadFile(request: DownloadManager.Request, @DownloadMeth
 
 fun Activity.isAutoRotateOn() = Settings.System.getInt(contentResolver, ACCELEROMETER_ROTATION, 0) == 1
 
+fun Fragment.isPackageInstalled(@ExternalPlayerPackage packageName: String) = try {
```

Comment:
I don't think the annotation is necessary here.
```suggestion
fun Fragment.isPackageInstalled(packageName: String) = try {
```

Reply:
Any specific reason? Or just because it is redundant? I thought it was good because due to the changes in Android 11 it is necessary to keep control of all the apps that we query in one place due to the new privacy limitations, unless in the future you want to add other apps apart from `@ExternalPlayerPackage`, in any case, if you confirm that you want this change, I will apply it.

Reply:
> Any specific reason? Or just because it is redundant? I thought it was good because due to the changes in Android 11 it is necessary to keep control of all the apps that we query in one place due to the new privacy limitations, unless in the future you want to add other apps apart from `@ExternalPlayerPackage`, in any case, if you confirm that you want this change, I will apply it.

I thought the function could be more generic, but the access control of Android 11 is a very good point, haven't thought of that. Let's merge this then. :+1: